src/Geboortezorg-3-2/Dev/XIS-Server/gbz-fhir3-0-2-izk-xis-2-2.xml

```diff
@@ -301,13 +301,6 @@
          <nts:with-parameter name="code" value="698350008"/>
          <nts:with-parameter name="system" value="http://snomed.info/sct"/>
       </nts:include>
-      <nts:include value="gbz-assert-returnCountWithCode">
```

Comment:
Dit is geen duplicate toch? Het verschil is dat hier enkel een count wordt gedaan en bij de content assert wordt echt de inhoud van de fixture gecheckt.

Reply:
AlcoholUse en TobaccaUse zijn verplaatst van 8a-ServeObservations naar 8b-ServeObservations. Ik denk dat er bij het mergen toch iets niet  goed is gegaan.